Userland/Makefile

```diff
@@ -12,25 +12,28 @@ INCLUDE_FLAGS = -I.. -I. -I../LibC
 DEFINES = -DSERENITY -DSANITIZE_PTRS -DUSERLAND
 
 CXXFLAGS = -MMD -MP $(WARNING_FLAGS) $(OPTIMIZATION_FLAGS) $(FLAVOR_FLAGS) $(ARCH_FLAGS) $(STANDARD_FLAGS) $(INCLUDE_FLAGS) $(DEFINES)
-CXX = i686-pc-serenity-g++
+CXX = i686-pca-serenity-g++
```

Comment:
Oops, looks like I missed a typo here. Fixed it in adc91d92a311e55ea5c50d23932e00a2d8c30d94